src/helpers/typography.test.js

```diff
@@ -273,8 +272,8 @@ describe('@mixin govuk-typography-responsive', () => {
     describe('and $important is set to true', () => {
       it('marks font size and line height as important', async () => {
         const sass = `
-          ${sassBootstrap}
           $govuk-typography-use-rem: false;
+          ${sassBootstrap}
```

Comment:
I have moved these around to better demonstrate how our users would write their SCSS